src/api/VoteMonitor.Api.Location/MappingProfile/PollingStationProfile.cs

```diff
@@ -19,6 +20,14 @@ public PollingStationProfile()
             CreateMap<UpdatePollingSectionCommand, PollingStationInfo>()
                 .ForMember(dest => dest.LastModified, c => c.MapFrom(src => DateTime.UtcNow));
 
+            CreateMap<AddPollingStationInfo, RegisterPollingStationCommand>()
+                .ForMember(dest => dest.CountyCode, c => c.MapFrom(src => src.CountyCode))
+                .ForMember(dest => dest.IdPollingStation, c => c.MapFrom(src => src.IdPollingStation))
+                .ForMember(dest => dest.IsPollingStationPresidentFemale, c => c.MapFrom(src => src.IsPollingStationPresidentFemale))
```

Comment:
I think it is not required to specify the mapping rules if you have a matching (name and type) property between source and destination